dist/undo.js

```diff
@@ -0,0 +1,654 @@
+(function (global, factory) {
```

Comment:
This file must have slipped in by mistake

Reply:
Yep! 😏